src/reporting/Market.sol

```diff
@@ -0,0 +1,604 @@
+        allReportersDisputeBondToken = DisputeBondToken(0);
+        tentativeWinningPayoutDistributionHash = 0;
+        return true;
+    }
+
+    ////////
+    //////// Helpers
+    ////////
+
+    function getReportingToken(uint256[] _payoutNumerators) public returns (ReportingToken) {
+        bytes32 _payoutDistributionHash = derivePayoutDistributionHash(_payoutNumerators);
+        ReportingToken _reportingToken = reportingTokens[_payoutDistributionHash];
+        if (address(_reportingToken) == NULL_ADDRESS) {
+            _reportingToken = ReportingTokenFactory(controller.lookup("ReportingTokenFactory")).createReportingToken(controller, this, _payoutNumerators);
+            reportingTokens[_payoutDistributionHash] = _reportingToken;
+        }
+        return _reportingToken;
+    }
+
+    function fundDisputeBondWithReputation(address _bondHolder, DisputeBondToken _disputeBondToken, uint256 _bondAmount) private returns (bool) {
+        require(_bondHolder == _disputeBondToken.getBondHolder());
+        reportingWindow.getReputationToken().trustedTransfer(_bondHolder, _disputeBondToken, _bondAmount);
+        return true;
+    }
+
+    function transferIncorrectDisputeBondsToWinningReportingToken() private returns (bool) {
+        require(isFinalized());
+        ReputationToken _reputationToken = reportingWindow.getReputationToken();
+        if (getBranch().getForkingMarket() == this) {
+            return true;
+        }
+        if (address(automatedReporterDisputeBondToken) != NULL_ADDRESS && automatedReporterDisputeBondToken.getDisputedPayoutDistributionHash() == finalPayoutDistributionHash) {
+            _reputationToken.trustedTransfer(automatedReporterDisputeBondToken, getFinalWinningReportingToken(), _reputationToken.balanceOf(automatedReporterDisputeBondToken));
+        }
+        if (address(limitedReportersDisputeBondToken) != NULL_ADDRESS && limitedReportersDisputeBondToken.getDisputedPayoutDistributionHash() == finalPayoutDistributionHash) {
+            _reputationToken.trustedTransfer(limitedReportersDisputeBondToken, getFinalWinningReportingToken(), _reputationToken.balanceOf(limitedReportersDisputeBondToken));
+        }
+        return true;
+    }
+
+    function derivePayoutDistributionHash(uint256[] _payoutNumerators) public constant returns (bytes32) {
+        uint256 _sum = 0;
+        require(_payoutNumerators.length == numOutcomes);
+        for (uint8 i = 0; i < numOutcomes; i++) {
+            require(_payoutNumerators[i] <= payoutDenominator);
+            _sum += _payoutNumerators[i];
+        }
+        require(_sum == payoutDenominator);
+        return sha3(_payoutNumerators);
+    }
+
+    function getReportingTokenOrZeroByPayoutDistributionHash(bytes32 _payoutDistributionHash) public constant returns (ReportingToken) {
+        return reportingTokens[_payoutDistributionHash];
+    }
+
+    ////////
+    //////// Getters
+    ////////
```

Comment:
So many slashes!  Recommend just:
```
//
// Getters
//
```
Also, we should be consistent with spacing, Helpers has an empty line after the comment, Getters does not.